### DISCUSSION TITLE
Returning the age of a particle

### DISCUSSION THREAD

**Discussion by LukeJMcCartin** [OpenDrift/opendrift#1443] [General]
I am interested in returning the age of a particle (i.e. the number of seconds since the particle was released). The function self.time_step.total_seconds() returns the time since the last time step; however, I don't know how to extract the time that has elapsed since the particle was released. 

Any help would be greatly appreciated!

**Comment by knutfrode** [comment]
Hi, there is a particle property named "age_seconds", see e.g. here: 
https://github.com/OpenDrift/opendrift/issues/1195

**Comment by LukeJMcCartin** [comment]
Thank you very much for your quick reply! I will try that.

**Comment by LukeJMcCartin** [comment]
This worked just as expected. Thank you again.
